docs/learn/guides/service_orchestration/src/appointment_service/main.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/http;
+
+http:Client reservationEP = new ("http://localhost:8081/reservation");
+http:Client paymentEP = new("http://localhost:8082/payment");
+
+@http:ServiceConfig {
+    basePath: "/doctorAppoinment"
+}
+service doctorAppoinment on new http:Listener(9090) {
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/reservation"
+    }
+    resource function makeReservation(http:Caller caller, http:Request request) {
+        json requestPaylod = checkpanic request.getJsonPayload();
```

Comment:
Instead of _checkpanic_ can we handle the error by throwing it and/or logging it appropriately in all the locations?

Reply:
This is a simple guide. To reduce the complexity and for the ease of understanding, we have added checkpanics.